src/app/nada-consta/list/nada-consta-list.component.spec.ts

```
@@ -190,14 +214,304 @@ describe('NadaConstaListComponent', () => {
+
+      const mockEvent = new Event('click');
+      component.openOptions(mockEvent, 999);
+
+      expect(component.contextMenuItems).toHaveLength(0);
+      expect(actionsMenuMock.toggle).not.toHaveBeenCalled();
+    });
+
+    it('deve executar comando Imprimir ao clicar', fakeAsync(() => {
+      const row = { id: 20, status: 'COMPLETED', usuario: { nome: 'José' } } as any;
+      component.objects = [row];
+      const imprimirSpy = jest.spyOn(component, 'imprimirNadaConsta').mockImplementation(() => {});
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)